rt/rs/client/src/main/java/org/apache/cxf/jaxrs/client/ClientProxyImpl.java

```diff
@@ -285,7 +285,8 @@ private static boolean isIgnorableParameter(Method m, Parameter p) {
             return true;
         }
         return p.getType() == ParameterType.REQUEST_BODY
-            && m.getParameterTypes()[p.getIndex()] == AsyncResponse.class;
+            && (m.getParameterTypes()[p.getIndex()] == AsyncResponse.class
```

Comment:
this would make it so that any client proxy can use `InvocationCallback`.  Is it worth adding sys tests that verify this?

Reply:
Good point.  This probably should be isolated to just MP Rest Clients.  I'll try to refactor this to avoid impacting non-MP proxy clients.